app/controllers/images_controller.rb

```
@@ -29,6 +29,12 @@ def create
         end
 
     end
+    
+    def destroy
+        @image = Image.find(params[:id])
+        @image.destroy
```

I'd highly recommend using `destroy!`, because if it fails, it will raise an exception. This might seem bad -- but it makes the failure more explicit.

Currently, if line 35 fails, it will fail silently.